Lib/test/test_shutil.py

```diff
@@ -1825,6 +1825,11 @@ def test_move_file_to_dir(self):
         # Move a file inside an existing dir on the same filesystem.
         self._check_move_file(self.src_file, self.dst_dir, self.dst_file)
 
+    def test_move_file_to_dir_pathlike(self):
+        # Move a file to another location on the same filesystem.
+        src = pathlib.Path(self.src_file)
```

Comment:
Could you add a corresponding test for a `Path` object as the `dst`? It's important to keep it tested so no one breaks the behavior accidentally in the future.

Reply:
Yep it's not under test now and free to regress so great call.

Done and passing!